clickhouse_plantuml/table.py

```diff
@@ -63,6 +63,15 @@ def __init__(
         self.sampling_key = sampling_key
         self.columns = []  # type: List[Column]
 
+    def parse_additional_dependencies(self):
+        """
+        Add target table of MaterializedView to dependencies
+        """
+        if self.engine == 'MaterializedView':
+            match = re.search(re.compile(r"^CREATE MATERIALIZED VIEW \S+ TO (\S+)"), self.create_table_query)
```

Comment:
Please, take a look at the tests result in https://github.com/Felixoid/clickhouse-plantuml/pull/10/checks?check_run_id=1623259664#step:6:35